src/components/Table.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import moment from "moment";
+import GreenMark from "./StatusMarks/GreenMark";
+import RedMark from "./StatusMarks/RedMark";
+import NeutralMark from "./StatusMarks/NeutralMark";
+
+export default function Table() {
+  const habitsNamesHead = [
+    "Навык",
+    "Отслеживание привычки",
+    "Чтение книги",
+    "Спортзал",
+    "Ранний подъем",
+    "Потенциал дня"
+  ];
```

Comment:
I would recommend moving this list of habits name out of component as they are statically known (i.e. hardcoded). In the future, when your component will re-render, this const will be re-declared